aten/src/ATen/Parallel.cpp

```diff
@@ -14,7 +15,41 @@ namespace at {
 
 namespace {
 // Number of threads set by the user
-std::atomic<int> num_threads(-1);
+std::atomic<int> num_threads{-1};
+
+// Number of inter-op threads set by the user;
```

Comment:
Define this magical numbers as constants please. Or much better wrap into struct, as num_interop_threads.compare_exchange_strong(no_value, nthreads) night me misused somewhere and you will never know.

Reply:
I use anonymous namespace to limit visibility but I can wrap it into struct sure

Reply:
Yes please if it is not too much work. Otherwise it is error prone.